### ISSUE TITLE
ld: warning: direct access in function

### ISSUE THREAD

**Issue by majetideepak** [facebookincubator/velox#1592]
I recently updated my local dependencies by running `setup-macos.sh`. I now see a lot of warnings similar to the one below
```
ld: warning: direct access in function 'fmt::v7::appender fmt::v7::detail::write<char, fmt::v7::appender, unsigned __int128, 0>(fmt::v7::appender, unsigned __int128, fmt::v7::basic_format_specs<char> const&, fmt::v7::detail::locale_ref)' from file '/usr/local/lib/libfmt.a(format.cc.o)' to global weak symbol 'fmt::v7::detail::basic_data<void>::prefixes' from file '../../common/caching/libvelox_caching.a(AsyncDataCache.cpp.o)' 
```

**Comment by majetideepak** [comment]
cc: @mbasmanova

**Comment by kgpai** [comment]
@majetideepak @mbasmanova This pr should have fixed issue #2133 (requires you to rerun setup-macos.sh). Reopen if you still see the problem.
